README.md

```diff
@@ -1,4 +1,7 @@
 # Nextcloud Files Retention App
+
+[![REUSE status](https://api.reuse.software/badge/github.com/nextcloud/files_retention)](https://api.reuse.software/info/github.com/nextcloud/files_retention)
```

Comment:
Unregistered, @AndyScherzinger which details should be used when registering?

Reply:
Dropped you a message and did it for this one too 👍